### ISSUE TITLE
caps or lowercase when inline citing caption? [request]

### ISSUE THREAD

**Issue by RMHogervorst** [adletaw/captioner#30]
Love this package! one small request: Could you make two ways to cite?

display = "cite"  === small, exmple: As we can see in figure 1, pirates are a cause for global warming.

display = "Cite" === caps, example:  Figure 1 clearly shows that pirates and global warming follow the same trend.

**Comment by RMHogervorst** [comment]
I gues you could change 
https://github.com/adletaw/captioner/blob/master/R/captioner.R#L256-L258

to

```
  else if(display == "Cite" || display == "C")
    {
return(paste0(prefix, obj_num))
}
else if(display == "cite" || display == "c")
{
return(paste0(tolower(prefix), obj_num))
}

```

**Comment by adletaw** [comment]
Hello @RMHogervorst ,

Thanks for the suggestion!  I will certainly consider it, but from what I have seen figures are usually cited using the capital letter "Figure 1", is it not so?  It would definitely add some flexibility to the package though.

**Comment by RMHogervorst** [comment]
Yeah the caption is often with a capital,  but I describe figures without.

Op 12 sep. 2016 20:22 schreef "A.D. Letaw" <notifications@github.com>:
